ansible/playbooks/roles/upgrade/tasks/kubernetes.yml

```diff
@@ -1,25 +1,65 @@
 ---
+# During HA control plane upgrade server address in kubeconfig is switched to local for
+#   * compatibility between client and server versions
+#   * identifying correct server version
+
+- name: k8s/master | Switch apiserver address to local
+  include_tasks: kubernetes/utils/set-local-apiserver.yml # sets kubectl_context_cluster
```

Comment:
This file uses `kubectl` so to be on the safe side I would consider placing `k8s | Include wait-for-kube-apiserver.yml` task at the beginning, as we had before (it was the first task).

Reply:
I moved Wait for kube-apiserver task at the beginning of the upgrade.